### ISSUE TITLE
Plugin fails for docker for version 17.06.2-ce

### ISSUE THREAD

**Issue by kelebra** [marcuslonnberg/sbt-docker#76]
Hi, thanks a lot for useful sbt plugin.

However after couple of docker clients update I get the following stacktrace on **MacOS using docker** `17.06.2-ce` and setup from the guide:

```bash
java.lang.RuntimeException: Could not parse Version from 17.06.2-ce
: `.' expected but `6' found
	at sbtdocker.DockerVersion$.parseVersion(DockerVersion.scala:26)
	at sbtdocker.DockerVersion$.apply(DockerVersion.scala:20)
	at sbtdocker.DockerTag$.apply(DockerTag.scala:17)
	at sbtdocker.DockerBuild$$anonfun$buildAndTag$1.apply(DockerBuild.scala:75)
	at sbtdocker.DockerBuild$$anonfun$buildAndTag$1.apply(DockerBuild.scala:74)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at sbtdocker.DockerBuild$.buildAndTag(DockerBuild.scala:74)
	at sbtdocker.DockerBuild$.apply(DockerBuild.scala:45)
	at sbtdocker.DockerBuild$.apply(DockerBuild.scala:24)
	at sbtdocker.DockerSettings$$anonfun$baseDockerSettings$1.apply(DockerSettings.scala:17)
	at sbtdocker.DockerSettings$$anonfun$baseDockerSettings$1.apply(DockerSettings.scala:10)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

```bash
Oleksiis-MBP:docker kelebra$ docker --version
Docker version 17.06.2-ce, build cec0b72
```

Moreover I have the same issue on **Travis** which uses version `17.03.2-ce`.

**Comment by kelebra** [comment]
Seems like I found an issue. I was using plugin of version `1.4.0` and did not update to `1.4.1` because of the following note in `README`:

> Version 1.4.1 have been released for sbt 1.0.

After update to `1.4.1` things look fine again 👍 Just a heads up to anyone who was using `1.4.0` and is going to update to latest docker `17.03.2-ce`, you also need to update plugin to latest version.

I might contribute a humble PR with the following minor enhancements:

- Additional unit tests to parse multiple docker versions
- More expressive release note in `README`

Please let me know what you think about it, cheers!

**Comment by marcuslonnberg** [comment]
Added a note about this in the readme.
